detox/src/ios/expect.js

```diff
@@ -185,6 +186,14 @@ class ScrollColumnToValue extends Action {
   constructor(column,value) {
     super();
     this._call = invoke.callDirectly(GreyActions.actionForSetPickerColumnToValue(column,value))
+    this._call = invoke.callDirectly(GreyActions.actionForSetPickerColumnToValue(column, value));
```

Comment:
I think this line was duplicated by mistake?

Reply:
📗 ` [fe3a615](/wix/Detox/pull/1148/commits/fe3a6152e98f1d8fe352c78fde9ded4fe7d2168e)`